Composer/packages/client/src/pages/language-generation/code-editor.tsx

```diff
@@ -2,45 +2,101 @@
 // Licensed under the MIT License.
 
 /* eslint-disable react/display-name */
-import React, { useState, useEffect } from 'react';
-import { LgEditor } from '@bfc/code-editor';
+import React, { useState, useEffect, useMemo, useContext } from 'react';
+import { LgEditor, LGOption } from '@bfc/code-editor';
 import get from 'lodash/get';
 import debounce from 'lodash/debounce';
 import isEmpty from 'lodash/isEmpty';
-import { CodeRange, LgFile } from '@bfc/shared';
+import { Diagnostic } from 'botbuilder-lg';
+import { LgFile } from '@bfc/shared';
 
+import { StoreContext } from '../../store';
 import * as lgUtil from '../../utils/lgUtil';
 
 interface CodeEditorProps {
   file: LgFile;
-  onChange: (value: string) => void;
-  codeRange?: Partial<CodeRange> | null;
+  template: lgUtil.Template | null;
 }
 
+const lspServerPort = process.env.LANGUAGE_SERVER_PORT ? Number(process.env.LANGUAGE_SERVER_PORT) : 5000;
+const lspServerPath = '/lgServer';
```

Comment:
can we change the name of this endpoint to be something other than `lgServer` ?

how about `/language-server`

Reply:
we may have /luServer later. so how about 
/lg-language-server
/lu-language-server

or 
/language-server/lg
/language-server/lu